src/modules/user/controller/index.js

```diff
@@ -38,6 +41,47 @@ async function registerUser(req, res) {
   res.status(201).send({ statusCode: 'CREATED', user });
 }
 
+async function loginUser(req, res) {
+  const { email, password } = req.body;
+  const user = await User.findOne({
+    where: {
+      email: email
+    }
+  });
+
+
+  if (!user) {
+    return res.status(400).json({
+      statusCode: 'BAD_REQUEST',
+      errors: {
+        email: [
+          'User does not exist'
+        ]
+      }
+    });
+
+  }
+
+  const passwordMatches = await bcrypt.compare(password, user.password)
+  if (!passwordMatches) {
+    return res.status(400).json({
+      statusCode: 'BAD_REQUEST',
+      errors: {
+        password: [
+          'Passwords do not match'
```

Comment:
Giving this message at login is exploiting that the username exists in our db giving room for hackers.

Would you change the message to `Invalid Credentials` instead

Reply:
- [ ] @Best-Verie  you don't have to be specific to the error message

Reply:
yeah